File: app/src/main/java/com/drkein/stayon/tools/Pref.java
package com.drkein.stayon.tools;

import android.content.Context;

/**
 * Created by kein on 2017. 2. 22..
 */

public class Pref {
    private static final String TAG = Pref.class.getSimpleName();
    public static void setServiceRunning(Context ctx, boolean running) {
        ctx.getSharedPreferences(TAG, Context.MODE_PRIVATE).edit().putBoolean("running", running).apply();
    }

    public static boolean getServiceRunning(Context ctx) {
        return ctx.getSharedPreferences(TAG, Context.MODE_PRIVATE).getBoolean("running", false);
    }
}

File: app/src/main/java/com/drkein/stayon/tools/Pref.kt
package com.drkein.stayon.tools

import android.content.Context

/**
 * Created by kein on 2017. 2. 22..
 */

object Pref {
    private val TAG = Pref::class.java.simpleName
    @JvmStatic
    fun setServiceRunning(ctx: Context, running: Boolean) {
        ctx.getSharedPreferences(TAG, Context.MODE_PRIVATE).edit().putBoolean("running", running)
            .apply()
    }
    @JvmStatic
    fun getServiceRunning(ctx: Context): Boolean {
        return ctx.getSharedPreferences(TAG, Context.MODE_PRIVATE).getBoolean("running", false)
    }
}
